module/zfs/spa_stats.c

```diff
@@ -28,12 +28,12 @@
 /*
  * Keeps stats on last N reads per spa_t, disabled by default.
  */
-int zfs_read_history = 0;
+int zfs_spa_read_history = 0;
```

Comment:
Since this is tracking the history of `arc_read()` calls let's rename it `zfs_arc_read_history()` if we must rename it.